site/variants/dualeditor.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Stacks Editor</title>
+    <link rel="icon"
+        href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🐸</text></svg>" />
+    <link rel="stylesheet" href="https://unpkg.com/@stackoverflow/stacks/dist/css/stacks.min.css" />
+    <script src="https://unpkg.com/@stackoverflow/stacks/dist/js/stacks.min.js"></script>
+</head>
+
+<body class="p24">
+    <form class="wmx6 m-auto">
+        <textarea id="content" name="content" class="d-none"></textarea>
+        <h1 class="mt32">Dual Editor Playground</h1>
+        <div class="grid">
+            <div id="example-1" class="mr8"></div>
```

Comment:
Nit, completely up to you: I think it looks nicer if the editors are stacked on top of each other:

```suggestion
        <div class="grid fd-column">
            <div id="example-1" class="mb8"></div>
```